classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Getters and Setters for `@rentals` should be present, rentals is an `attr_accessor`, please kindly change it 🙃 🙃